campaigns/rustreach/how-I-came-to-rust.md

```diff
@@ -0,0 +1,58 @@
+
+But there were two things I found very notable:
+
+* The openness of the discussion and the ease of participation
+* The air around the community to build something to _use_
+
+Rust was always the attempt to apply techniques instead of coming up with their own (obviously, the process means that you need to come up with your own things). It was interesting to watch the process and rare to see that. I enjoyed watching and coding along.
+
+## Rust Berlin
+
+In 2014, I got in touch with @johannh about running a meetup. We were both curious about Rust, but wanted to get in touch with other people who were interested. So we formed Rust Berlin and quickly thereafter the Rust Hack & Learn, where - instead of coming and listening to a talk - you could pass by and hack on your own project. The Hack & Learn is still running every other week.
+
+We later went on to co-found RustFest with others.
+
+## Joining the Rust project
+
+There's a difference between programming Rust and being involved in the Rust project. For me, the difference is that project work somewhat goes above the sphere of your personal projects. Rust Berlin, I wouldn't consider a part of my project work. Helping other people out with my experiences, I do. Shortly after the 1.0 release, Steve contacted me whether I want to take part in the newly formed community team.
+
+I was very impressed by this. Most projects don't have an explicit community team. Sure, there's community workers, but they are rarely integrated into the project on an offical basis. Being recognised for my previous experience in that area and my opinions on a project scale had never happened to me, though. Also, the ability to think about FOSS community issues in a far wider scale than I had before was just too teasing to be true. I accepted :).
+
+## Why I'm staying
+
+I guess you won't be able to draw that much from that story. All the conditions might not apply to you. So I guess it's important to tell you why I am still around. And that's because I think the Rust project gets a lot of things _right_. Some of these things weren't always right and there's always things that need to be fixed. A previous draft of this text had included some of the things that were broken. I decided to remove them, because they are _fixed_. For me, Rust - the project - is based on a couple of things:
+
+* Work-focus. Steves invitation mail to me contained the sentence "we need to productively move forward". That hit the right note. Working on the Rust project is not a merit badge for me, it's a place where I want to spend my effort for a while.
```

Comment:
I like how you have a short phrase ("Work-focus") to summarise the point to come here. Just a minor suggestion but perhaps you could style these first words for each point in bold? I think that might make them stand out and make the list a bit easier to parse as a whole. (Please feel free to ignore any of these though, I know I'm making lots of comments!)